package.json

```diff
@@ -29,7 +29,7 @@
         "lint:root": "pnpm prettier --check './!(demo|docs|packages|storybook)/**/*.{js,json,md,yml,yaml}' && $npm_execpath cspell .changeset/*.md",
         "lint:eslint": "pnpm --recursive run lint:eslint",
         "lint:tsc": "pnpm --recursive run lint:tsc",
-        "lint:knip": "knip --exports --files --dependencies --tags=-knipignore",
+        "lint:knip": "pnpm --recursive run api-generator && knip --exports --files --dependencies --tags=-knipignore",
```

Comment:
needed to avoid
```
[log] Unused exported types (2)
[log] NewsStatus    enum  demo/api/src/news/entities/news.entity.ts:20:13
NewsCategory  enum  demo/api/src/news/entities/news.entity.ts:26:13
```